src/functionalTest/java/uk/gov/hmcts/reform/em/hrs/ShareScenarios.java

```diff
@@ -21,6 +22,7 @@
 import static org.hamcrest.Matchers.not;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
+@Ignore
```

Comment:
NOt sure again why we are Ignoring the tests?

Reply:
removed